examples/tts/conf/magpietts/magpietts_lhotse.yaml

```diff
@@ -87,6 +87,7 @@ model:
       shuffle: true
       num_workers: 6
       pin_memory: true
+      reweight_temperature: [1.0, 0.0]  # Temperature for re-weighting datasets. 1 is a neutral value. Lower temperature over-samples smaller datasets, and vice versa.
```

Comment:
I think you also need to add this to `dataloader.py` in `LhotseDataLoadingConfig` otherwise it won't get propagated to `cutset.py` functions. We should also have a test that uses this option through `get_lhotse_dataloader_from_config` to check it works.

Reply:
good point. indeed, my test using `get_lhotse_dataloader_from_config` failed to pass this new param to `propagate_attrs`. 

Fixed now.